tests/browserstack_automation/page_objects/signin.page.js

```diff
@@ -15,10 +15,10 @@ class SignIn extends Page{
     }
 
     get signInWithTwitterTextElement(){
-        return $('//span[text()="Sign in with Twitter"]');
+        return $('.csbvaL');
```

Comment:
Could you locate the element with an id instead of class name?

Reply:
Thanks for reviewing the changes, @fdiazq! I tried adding the ID for the "Sign in with Twitter" element, but  the WebApp running on localhost port 3000 is not displaying the Sign in with X/Twitter option on the SignIn page. As a result, I opted for the class name strategy, which is more stable .
Please have a look .